content/en/docs/reference/using-api/deprecation-guide.md

```diff
@@ -24,6 +24,14 @@ deprecated API versions to newer and more stable API versions.
 
 The **v1.26** release will stop serving the following deprecated API versions:
 
+#### Flow control resources {#flowcontrol-v126}
```

Comment:
nit:
```suggestion
#### Flow control resources {#flowcontrol-resources-v126}
```
to match fragment identifier `#webhook-resources-v122`

Reply:
done